rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -924,6 +929,96 @@ void saveFederationElection_shouldSaveInStorage() {
         assertArrayEquals(federationElectionSerializedWithVote, actualAbiCallElectionSerialized);
     }
 
+    @Nested
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+    @Tag("save fund transaction unsigned hash tests")
+    class SaveFundTransactionUnsignedHashTests {
+
+        @Test
+        void saveFundTransactionUnsignedHash_preLovell700_shouldNotSaveInStorage() {
+
+            // Arrange
+
+            ActivationConfig.ForBlock arrowheadActivations = ActivationConfigsForTest.arrowhead600().forBlock(0L);
+
+            StorageAccessor storageAccessor = new InMemoryStorage();
+            FederationStorageProvider federationStorageProvider = new FederationStorageProviderImpl(storageAccessor);
```

Comment:
Is it possible to move this to a setup method?

Reply:
Updated.